crates/ethereum/node/src/node.rs

```diff
@@ -348,7 +351,7 @@ where
             let blob_params = ctx
                 .chain_spec()
                 .blob_params_at_timestamp(current_timestamp)
-                .unwrap_or(ctx.chain_spec().blob_params.cancun);
+                .unwrap_or(BlobParams::cancun());
```

Comment:
Can we add a comment about this fallback? This fallback does make more sense to me than the previous code

Reply:
```suggestion
                .unwrap_or_else(BlobParams::cancun);
```

Reply:
this is fine we don't expect this to be used pre cancun anyway and with non default cancun config